core/chains/evm/config/chaintype/chaintype.go

```diff
@@ -23,6 +23,7 @@ const (
 	ChainZkEvm           ChainType = "zkevm"
 	ChainZkSync          ChainType = "zksync"
 	ChainZircuit         ChainType = "zircuit"
+	ChainDualBroadcast   ChainType = "dualBroadcast"
```

Comment:
What is this dual broadcast type chain? Is there more info on this new chain type?
Also, just curious, why should this be in the TXMv2 PR?